projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml

```diff
@@ -42,3 +42,8 @@ batch:
       env:
         variables:
           RELEASE_BRANCH: 1-25
+    - identifier: aws_bottlerocket_bootstrap_1_26
```

Comment:
I think im a bit surprised by this. If we are skipping it, do we still want it to be included in the build specs?  I probably wrote the code that handles this, but dont remember if it is going to matter...

Also, we are skipping this one?

Reply:
yeah i skipped this one too since we wont have BR 1.26 builds for a while anyway & the presubmit was showing some odd errors: https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/1823/bottlerocket-bootstrap-tooling-presubmit/1623693594554535936

but this was before i added the 1..26 additional pause line for the image builder. not sure if that would affect this though